lib/jsonapi/authorization/authorizing_processor.rb

```diff
@@ -12,10 +12,11 @@ class AuthorizingProcessor < JSONAPI::Processor
       set_callback :remove_resource, :before, :authorize_remove_resource
       set_callback :replace_fields, :before, :authorize_replace_fields
       set_callback :replace_to_one_relationship, :before, :authorize_replace_to_one_relationship
-      set_callback :create_to_many_relationship, :before, :authorize_create_to_many_relationship
-      set_callback :replace_to_many_relationship, :before, :authorize_replace_to_many_relationship
-      set_callback :remove_to_many_relationship, :before, :authorize_remove_to_many_relationship
       set_callback :remove_to_one_relationship, :before, :authorize_remove_to_one_relationship
+      set_callback :create_to_many_relationships, :before, :authorize_create_to_many_relationships
+      set_callback :replace_to_many_relationships, :before, :authorize_replace_to_many_relationships
+      set_callback :remove_to_many_relationships, :before, :authorize_remove_to_many_relationships
+      set_callback :replace_polymorphic_to_one_relationship, :before, :authorize_replace_polymorphic_to_one_relationship
```

Comment:
This line is too long and will cause the linter to fail the build. Sorry about that 😥 

Maybe write it like like this?

```rb
set_callback(
  :replace_polymorphic_to_one_relationship,
  :before,
  :authorize_replace_polymorphic_to_one_relationship
)
```